main_test.go

```diff
@@ -55,6 +55,12 @@ func TestCompiler(t *testing.T) {
 		runPlatTests("cortex-m-qemu", matches, t)
 	})
 
+	if runtime.GOOS != "linux" {
```

Comment:
Does this also need to exclude `freebsd`?

Reply:
Heh, good point. I'll simply check for windows or darwin instead, to be conservative.